hubspot/list.go

```diff
@@ -204,3 +230,19 @@ func (c *Client) List(ctx context.Context, resource string, opts *ListOptions) (
 
 	return &resp, nil
 }
+
+// ListByNextLink retrieves a list of items by a next link.
+// It doesn't require specifying filters and resources manually.
+func (c *Client) ListByNextLink(ctx context.Context, nextLink string) (*ListResponse, error) {
+	req, err := c.newRequest(ctx, http.MethodGet, nextLink, nil)
+	if err != nil {
+		return nil, fmt.Errorf("create new request: %w", err)
+	}
+
+	var resp ListResponse
+	if err := c.do(req, &resp); err != nil {
+		return nil, fmt.Errorf("do request: %w", err)
```

Comment:
please change `do request ` to `execute request`

Reply:
https://github.com/conduitio-labs/conduit-connector-firebolt/pull/15#discussion_r963549628

Reply:
Done!